api/router.go

```diff
@@ -322,15 +322,15 @@ func getRoutes(debug bool) []routeHandler {
 }
 
 func wrapHandler(handler http.Handler, route routeHandler, dt *Dt) http.Handler {
-	h := headerMiddleware(handler, route.headers)
+	h := headerMiddleware(dtMiddleware(handler, dt), route.headers)
```

Comment:
Unrelated to this line number, but the commit message here could be better (e.g. WIP -> something else)